### DISCUSSION TITLE
Initialize the MASS database (now that Home Assistant can natively access NAS)

### DISCUSSION THREAD

**Discussion by FrankJaco** [music-assistant/support#1297] [Q&A] [answered]
I would like to completely wipe out the current MASS database and rebuild it as have NAS media configured.  How can I accomplish that?  (In the old days, deleting the MusicAssistant.DB file in "Config" would do that, but now with MASS as an AddOn, that no longer works.

Thanks,

fj

**Comment by marcelveldt** [comment, marked as answer]
Just reinstall the add-on

**Reply by marcelveldt** [reply]
There is also an advanced setting in the settings for the Music core module/controller.
